terraspace/app/stacks/peer-kubernetes-components/tfvars/us-west-2/dev.tfvars

```diff
@@ -0,0 +1 @@
+bitswap_peer_deployment_branch = "multienv" # TODO: Back to head when 'https://github.com/ipfs-elastic-provider/ipfs-elastic-provider-bitswap-peer-deployment/pull/4' is approved 
```

Comment:
I'd hold off merging this PR until we can point it back to `HEAD`

Reply:
Done!